variables.tf

```diff
@@ -51,7 +51,19 @@ variable "truefoundry_db_postgres_version" {
 
 variable "truefoundry_db_network_cidr" {
   type        = string
-  description = "Network CIDR for the truefoundry postgres database. Minimum range is /24. This CIDR must be different from the main subnet where your cluster is gettin created. This subnet is created inside the GCP's network."
+  description = "Network CIDR for the truefoundry postgres database. Minimum range is /24. This CIDR must be different from the main subnet where your cluster is getting created. This subnet is created inside the GCP's network."
+  default     = "10.0.0.0/24"
+
+  validation {
+    condition = (
+      var.truefoundry_db_enable == false ? true : (
+        var.truefoundry_db_network_cidr != null &&
+        can(regex("^([0-9]{1,3}\\.){3}[0-9]{1,3}/[0-9]{1,2}$", var.truefoundry_db_network_cidr)) &&
+        tonumber(split("/", var.truefoundry_db_network_cidr)[1]) <= 24
```

Comment:
`/24` is preferred but is a range of 256 IP address which is a lot wrt a DB, please keep it `/28`